images/jail/jail.dockerfile

```diff
@@ -71,10 +64,10 @@ RUN apt update && \
         numactl \
         htop \
         hwloc \
-        rdma-core \
-        ibverbs-utils \
-        libpmix2 \
```

Comment:
Isn't it needed due to `dev` package?

Reply:
Thats not a real package:
```bash
aptitude show libpmix2
No candidate version found for libpmix2
Package: libpmix2
State: not a real package
Provided by: libpmix2t64 (5.0.1-4.1build1)
```

And we installing libpmix-dev which has dependency
```bash
aptitude show libpmix-dev
Package: libpmix-dev
Version: 5.0.1-4.1build1
State: installed
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Architecture: amd64
Uncompressed Size: 11.3 M
Depends: libpmix2t64 (= 5.0.1-4.1build1), libevent-dev, libhwloc-dev, zlib1g-dev
```